enterprise/admin/views.py

```diff
@@ -229,28 +232,47 @@ def _enroll_users(cls, enterprise_customer, emails, course_id, mode, request):
             course_id: The ID of the course in which we want to enroll
             mode: The enrollment mode the users will be enrolled in the course with
             request: The HTTP request the enrollment is being created by
+            notify: Whether to notify (by email) the users that have been enrolled
         """
         enrolled = []
         non_existing = []
         failed = []
         enrollment_client = EnrollmentApiClient()
+        course_details = CourseApiClient().get_course_details(course_id)
+        reverse_method = get_reverse_by_site(request, enterprise_customer.site)
+        course_url = reverse_method('about_course', args=(course_id,))
```

Comment:
This `get_reverse_by_site` API seems needlessly complicated to me. Can't you just create a `reverse_site_url(request, site, *args, **kwargs)` method?

```python
course_url = reverse_site_url(request, site=enterprise_customer.site, view_name='about_course', args=(course_id, ))
```

You can also implement it simply as `return 'https://' + site.domain + reverse(*args, **kwargs)`.

Actually, on second thought, this assumes that the enterprise app and the LMS are deployed into the same django project. Does the Course catalog API not provide the URL of the course?